docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   # Database Services
   # ================================
   postgres:
-    image: postgres:latest
+    image: postgres:18.0-alpine3.22
```

Comment:
@mazzy89, out of curiosity, why not latest and instead a specific PostgreSQL version?

Reply:
18.0 is the latest at the time we are speaking. Nevertheless pinning dpendencies is very good practice. Dependency was not pinned. As result it broke the volume mounting that in the last version of Docker Postgres image has changed as you can read [here](https://forums.docker.com/t/mysterious-docker-issue-with-postgres/149898/7).